Train Epoch: 1 [0/5248 (0%)]	Loss: 0.667262
Train Epoch: 1 [3200/5248 (61%)]	Loss: 0.486251
Average loss : 0.5767565965652466
model sent successfully!
Train Epoch: 1 [0/5248 (0%)]	Loss: 0.314616
Train Epoch: 1 [3200/5248 (61%)]	Loss: 0.327515
Average loss : 0.32106560468673706
model sent successfully!
Train Epoch: 1 [0/5248 (0%)]	Loss: 0.361427
Train Epoch: 1 [3200/5248 (61%)]	Loss: 0.253580
Average loss : 0.3075035959482193
model sent successfully!
Train Epoch: 1 [0/5248 (0%)]	Loss: 0.203316
Traceback (most recent call last):
  File "run.py", line 46, in <module>
    optimizer=optimizer, epoch=epoch)                ###logger added
  File "/home/nikamanth/Documents/fyp/src/client/train.py", line 9, in train
    for batch_idx, (data, target) in enumerate(train_loader): # <-- now it is a distributed dataset
  File "/home/nikamanth/anaconda3/envs/torch/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 345, in __next__
    data = self._next_data()
  File "/home/nikamanth/anaconda3/envs/torch/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 385, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/home/nikamanth/anaconda3/envs/torch/lib/python3.7/site-packages/torch/utils/data/_utils/fetch.py", line 44, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/nikamanth/anaconda3/envs/torch/lib/python3.7/site-packages/torch/utils/data/_utils/fetch.py", line 44, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/nikamanth/Documents/fyp/src/client/dataloader.py", line 51, in __getitem__
    image = PIL.ImageOps.grayscale(image)
  File "/home/nikamanth/anaconda3/envs/torch/lib/python3.7/site-packages/PIL/ImageOps.py", line 476, in grayscale
    return image.convert("L")
  File "/home/nikamanth/anaconda3/envs/torch/lib/python3.7/site-packages/PIL/Image.py", line 893, in convert
    self.load()
  File "/home/nikamanth/anaconda3/envs/torch/lib/python3.7/site-packages/PIL/ImageFile.py", line 265, in load
    n, err_code = decoder.decode(b)
KeyboardInterrupt
